core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/azureModels.scala

```diff
@@ -3,9 +3,12 @@ package org.broadinstitute.dsde.workbench.leonardo
 import java.util.UUID
 
 final case class WsmControlledResourceId(value: UUID) extends AnyVal
+final case class StorageContainerName(value: String) extends AnyVal
```

Comment:
maybe use `ContainerName` from the wb-libs AzureStorage here to cut down on duplicate models

Reply:
you mean `org.broadinstitute.dsde.workbench.google2.KubernetesSerializableName.ContainerName`? that one doesn't seem right...this is an Azure storage container

Reply:
This one https://github.com/broadinstitute/workbench-libs/blob/develop/azure/src/main/scala/org/broadinstitute/dsde/workbench/azure/AzureStorageService.scala#L66

Reply:
ah cool..I'll pull in the latest wb-libs